fdbclient/include/fdbclient/GetEncryptCipherKeys_impl.actor.h

```diff
@@ -217,6 +217,54 @@ Future<EKPGetBaseCipherKeysByIdsReply> _getUncachedEncryptCipherKeys(Reference<A
 	}
 }
 
+ACTOR template <class T>
```

Comment:
Thinking loud: is there a reason we don't want to keep this code into ClusterController? CC knows about EKP interface and seems like the best place for: EKP still exists and also is EKP reachable. Putting code in `GetEncryptCipherKeys` does feel strange and also bit counter-intuitive, imho
Thoughts?

Reply:
We could the only issue is no client would be able to use these APIs as the CC is kept in fdbserver namespace. Currently i believe the only client is Backup but that might change in the future? Regardless if we do it, should be a separate PR

Reply:
Trying to understand the above reasoning. My understanding is CC currently is invoked by Status actor to fetch required information? If yes, given we are appending to `status json` workflow, extending CC to fetch KMS health sounds like a logical extension. 
Also, why do we need all encryption clients to access this API? Are we proposing individual processes maintaining the health status?

Reply:
My main reasoning was all communication to the EKP from clients goes through actors in this file. We could do this in the CC but then im not sure if the code organization would make sense. Keeping it here feels better for code organization.

Reply:
Also with the amount of code already in CC im not a huge fan of adding more functionality there (would rather keep it here)

Reply:
GetEncryptCipherKey module is responsible for fetching encryption for a given encryption participant processes, however, the code here is responsible to report KSM communication status, collocating the code where Cluster status is kept seems more reasonable thing to me.